.github/dependabot.yml

```diff
@@ -1,6 +1,10 @@
 version: 2
 updates:
-  - package-ecosystem: "npm"
-    directory: "/"
-    schedule:
-      interval: "daily"
+- package-ecosystem: "npm"
+  directory: "/"
+  schedule:
+    interval: "daily"
+  groups:
+    typespec:
+      patterns:
```

Comment:
What exactly is this doing? Is it limiting daily updates to only typespec packages? Will it still update other packages?

Reply:
It ensures updates of packages matching `*typespec*` will be updated in a single grouped PR, rather than the default of one PR per package.  Packages not matching `*typespec*` will continue to be updated in individual PRs.